sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableConfig.java

```diff
@@ -159,13 +159,33 @@ void validate() {
         "Could not obtain Bigtable instance id");
   }
 
+  /**
+   * @param <T> parameter The runtime parameter.
+   * @return {@code null} if the input parameter is {@code null}, return the String value of runtime
+   *     parameter if the parameter is accessible, returns "Unavailable during pipeline
+   *     construction" if the runtime parameter is not accessible for debugging purpose.
+   */
+  @Nullable
+  static <T> String getDisplayDataForParameter(@Nullable ValueProvider<T> parameter) {
+    if (parameter == null) return null;
+
+    if (parameter.isAccessible()) {
+      return String.valueOf(parameter.get());
+    }
+    return "Unavailable during pipeline construction";
+  }
+
   void populateDisplayData(DisplayData.Builder builder) {
     builder
         .addIfNotNull(
-            DisplayData.item("projectId", getProjectId()).withLabel("Bigtable Project Id"))
+            DisplayData.item("projectId", getDisplayDataForParameter(getProjectId()))
+                .withLabel("Bigtable Project Id"))
+        .addIfNotNull(
+            DisplayData.item("instanceId", getDisplayDataForParameter(getInstanceId()))
```

Comment:
I'm not sure why we had to update display-data related logic for this PR (fix validation issue). Should these updates be a separate PR ?

Reply:
I can move this to a separate PR.

Reply:
I created a separate PR for disabling validation: https://github.com/apache/beam/pull/6413.